modules/reports/client-react/pdf/client/containers/WithExportPDF.tsx

```diff
@@ -0,0 +1,22 @@
+import React, { Fragment, Component } from 'react';
+import ReactToPrint from 'react-to-print';
```

Comment:
why use 'react-to-print' library, can use native browser functionality 'window.print', the same result.